operacje_na_listach/Function_to_reject.py

```diff
@@ -0,0 +1,7 @@
+def reject(list):
```

Comment:
this name must be more specific - `reject_odd_numbers` or something like that

Reply:
name of file or function?